pkgs/development/python-modules/glyphslib/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, stdenv
+, setuptools
+, autoPatchelfHook
+, fonttools
+, openstep-plist
+, ufoLib2
+, unicodedata2
+}:
+
+buildPythonPackage rec {
+  pname = "glyphslib";
+  version = "6.0.4";
+
+  format = "wheel";
+
+  src = fetchPypi{
+    pname = "glyphsLib";
+    inherit version format;
+    dist = "py3";
+    python = "py3";
+    sha256 = "zoiNIISxdJIjQQncRse1tJ3Gg+9Tgku8K+GUzQ8w8iQ=";
+  };
```

Comment:
We could have merged this if you didn't change this to wheel. We where building from source before which is always preferable. Why do we want to patch the wheel if we can build it?